app/controllers/quotes_controller.rb

```diff
@@ -1,26 +1,18 @@
 class QuotesController < ApplicationController
-
-
   def create
-
-    @quote = Quote.create(quote_params)
```

Comment:
`create` is `new` then `save` so the block here is repeated by calling `save` twice